go.mod

```diff
@@ -1,23 +1,41 @@
 module github.com/ThreeDotsLabs/watermill
 
 require (
+	cloud.google.com/go v0.34.0
```

Comment:
can you run `make generate_gomod`? It should clean up it and get the newest versions :wink:

Reply:
because go mod may failed to resolve dependencies due to pinned dependencies and etc.
```
go list -m -u -v 
go mod vendor
go mod tidy
```